modules/installation-three-node-cluster.adoc

```diff
@@ -39,7 +54,13 @@ compute:
 ====
 You must set the value of the `replicas` parameter for the compute machines to `0` when you install {product-title} on user-provisioned infrastructure, regardless of the number of compute machines you are deploying. In installer-provisioned installations, the parameter controls the number of compute machines that the cluster creates and manages for you. This does not apply to user-provisioned installations, where the compute machines are deployed manually.
 ====
-
+ifdef::ibm-z,ibm-z-kvm[]
++
+[NOTE]
+====
+A minimum of 21 GB per control plane node is preferred.
```

Comment:
can we change the note to:

"The minimum for control plane nodes is 21 GB. "

Reply:
this would be better, as it explains the 21 GB. 

"The minimum for control plane nodes is 21 GB. For three control plane nodes this is the memory equivalent of a minimum five node cluster. "